testador/zzcinemais.sh

```diff
@@ -0,0 +1,71 @@
+$ zzcinemais 
+9 - Uberaba  - MG
+11 - Patos de Minas - MG
+21 - Guaratinguetá - SP
+32 - Anápolis - GO
+34 - Montes Claros - MG
+35 - Shop. Alameda - Juiz de Fora - MG
+36 - Ituiutaba - MG
+37 - Araxá - MG
+38 - Lorena - SP
+39 - Jardim Norte - Juiz de Fora - MG
+$ zzcinemais 9 | sed '/ - [A-Z][A-Z]$/s/.*/CIDADE/;/\/[0-9]\{4\}$/s/.*/DATA/;/[0-2][0-9]h[0-5][0-9]/s/.*/HORARIO/;/^$/s/^/AA/;/[^A-Z[^A-Z]/{s/.*/FILME/;}' | sort | uniq
```

Comment:
E está errado mesmo. Faltou fechar primeira a lista.
Valeu de novo pela força camarada!